client/components/dialog/index.jsx

```diff
@@ -59,15 +59,15 @@ var Dialog = React.createClass( {
 			transitionName = this.props.baseClassName || 'dialog';
 
 		ReactDom.render(
-			<SingleChildCSSTransitionGroup
+			<ReactCSSTransitionGroup
 				transitionName={ transitionName }
 				component="div"
 				transitionLeave={ this.props.transitionLeave }
-				onChildDidLeave={ this.onDialogDidLeave }
-				enterTimeout={ this.props.enterTimeout }
-				leaveTimeout={ this.props.leaveTimeout }>
+				componentDidLeave={ this.onDialogDidLeave }
```

Comment:
I'm having a bit of trouble pin-pointing whether this will have the same effect as `onChildDidLeave`. I see it's documented on [React's Animation page](https://facebook.github.io/react/docs/animation.html), though it is [not passed](https://github.com/facebook/react/blob/713401f2eb4b892f872ec61b15fdb597c4eeaae6/src/addons/transitions/ReactCSSTransitionGroup.js#L72-L84) the same way as [in our own variant](https://github.com/Automattic/wp-calypso/blob/daae100a75d99aafd5d247b8f84f77531dc626a8/client/components/single-child-css-transition-group/index.js#L33-L42). Since this is still used in the reader, we'll want to ensure that it behaves the same way as it did previously, or refactor the Reader to not depend on this behavior.

See also: 903fd0ed9ad0196fc3907ff165f5bd21d98a9974